integration-test-groups/aws2/aws2-s3/src/test/java/org/apache/camel/quarkus/component/aws2/s3/it/Aws2S3Test.java

```diff
@@ -41,6 +44,9 @@
     private int objects_num_before;
     private int objects_num_after;
 
+    @Aws2Client(LocalStackContainer.Service.S3)
```

Comment:
We inject an Aws2Client that is not used elsewhere. Is there a trick to be explained here ?